examples/bpk-component-accordion/stories.js

```diff
@@ -73,3 +73,8 @@ export const WithoutDivider = SingleItemExampleWithoutDivider;
 
 export const VisualTest = SingleItemExample;
 export const VisualTestOnDark = WithDarkBackgroundExample;
+
+export const VisualTestWithZoom = VisualTest.bind({});
+VisualTestWithZoom.args = {
+  fontSize: '32px'
```

Comment:
Might we want with this to possibly make it easier to control vs individually change the font size (should we need to) do it where we use one of our font tokens instead that way we can always use something that could change across the board?

Reply:
not sure if this is what you mean here but I think a variable would be more flexible if in the future this value needs change

Reply:
I'm not sure I understand the comment/s 😄
But the value will not need to change in the future, unless the WCAG change so that instead of 200% zoom we need to support something else. We could use 2 *`fontSizeBase` so that it's more obvious it represents 200% zoom?

Reply:
It might make it clear to use values at least from our system so its clear its a number we came up with and self documenting, just from past experience of just having a number of question raised were 'how did we get to this number?' or 'can it be documented how we got to this 'magic number'?'

Reply:
I don't really want to link it to our system, nor add a comment for every story that uses it, let me think how to document this one 🤔 🧐